src/Util.php

```diff
@@ -75,4 +75,71 @@ public static function getTimestamp() {
         $micro = sprintf("%06d", ($t - floor($t)) * 1000000);
         return date('Y-m-d\TH:i:s.' . $micro . 'O', $t);
     }
+
+    /**
+     * determine which language out of an available set the user prefers most
+     *
+     * @method preferedLanguage returns the client's preferred language from a list of options.
+     * @param $available_languages {Array} list of language-tag-strings (must be lowercase) that are available
+     * @param $http_accept_language {String} a HTTP_ACCEPT_LANGUAGE string
+     *   (read from $_SERVER['HTTP_ACCEPT_LANGUAGE'] if left out)
```

Comment:
I suspect that most IDEs will be able to interpret this form of docblocks thus failing to provide proper autocompletion suggestions. Something like the following instead will work for sure:

```php
    /**
     * determine which language out of an available set the user prefers most
     *
     * @method preferedLanguage returns the client's preferred language from a list of options.
     * @param string[] $available_languages  list of language-tag-strings (must be lowercase) that are available
     * @param string   $http_accept_language a HTTP_ACCEPT_LANGUAGE string
     *   (read from $_SERVER['HTTP_ACCEPT_LANGUAGE'] if left out)
     */
```